### ISSUE TITLE
Different log file for each manager process

### ISSUE THREAD

**Issue by peastman** [MolSSI/QCFractal#767]
In the [documentation on setting up a compute manager](https://molssi.github.io/QCFractal/admin_guide/managers/index.html#compute-manager-setup-for-an-hpc-cluster), it says to specify the log file with

```
logfile: qcfractal-manager.log
```

I'm going to be having a whole lot of managers all running at once on different nodes of a cluster.  I want each one to write to a different log file.  Since some of the other fields seem to support environment variables, I was hoping I could just write

```
logfile: "qcfractal-manager-${SLURM_JOBID}.log"
```

but that doesn't work.  Is there an easy way to make each job use a different log file?

**Comment by dotsdl** [comment]
To get a log file per job, what I've done on various HPC clusters is to use `envsubst` to create a config file specific for that job, with environment variables like you have here substituted in. That config file is then used by the compute manager started by the job.

Something like:

```bash
envsubst < config.yml > configs/config.${SLURM_JOBID}.yaml

qcfractal-compute-manager --config configs/config.${SLURM_JOBID}.yaml
```

**Comment by bennybp** [comment]
The way outlined above is probably the best you could do now. You could also automatically create directories and copy a config, and have a relative path to a logfile there.

I did fix this for the future in #768 , so thanks for the suggestion :)

**Comment by peastman** [comment]
That works.  Thanks!
